tolerance-health/apps/quarkus-calculator/src/main/docker/Dockerfile.native

```diff
@@ -14,7 +14,7 @@
 # docker run -i --rm -p 8080:8080 quarkus/expense-validator
 #
 ###
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6
```

Comment:
We need to add this image to https://github.com/RedHatTraining/DO378/blob/main/classroom/osp/inventory/group_vars/all/vars.yml

I'm approving this PR and will put the reminder in the DO378 repo.